students/Adolphe_N/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Did you add this .project file on purpose? there is some debate about whether this kind of IDE file should be in source control...

if you son't want to manage them with git, you can do:

`git rm .project .pydevproject`

Reply:
No it was not on purpose! i think i might have generated them while i was trying to make python 3.7 interpreter works because i had 2.7 already installed. i will remove them. Thanks

Reply:
No it was not on purpose! i think i might have generated them while i was trying to make python 3.7 interpreter works because i had 2.7 already installed. i will remove them. Thanks